repos/fdbt-types/matchingJsonTypes.ts

```diff
@@ -237,6 +237,13 @@ export interface ReturnTicket extends BasePointToPointTicket {
     inboundFareZones: FareZone[];
     outboundFareZones: FareZone[];
     returnPeriodValidity?: ReturnPeriodValidity;
+    additionalServices?: AdditionalService[];
```

Comment:
Should this be SelectedService[] as we have that above?

Reply:
used SelectedService